Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PHP_VERSION=7.2
```

Comment:
The only files with Docker configuration are Dockerfile and docker-compose.yml.  Adding these to .gitattributes is fine but creating a directory for the Dockerfile isn't necessary.

Reply:
If we eventually add _more_ assets, however, we'll want to create that directory. I'd rather create it now so that once that happens, existing contributors are not then confused about where the files moved.